app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -3,9 +3,27 @@ module VerifyProfileConcern
 
   def account_or_verify_profile_url
     return reactivate_account_url if user_needs_to_reactivate_account?
-    return account_url unless profile_needs_verification?
-    return idv_gpo_url if gpo_mail_bounced?
-    idv_gpo_verify_url
+    if profile_needs_verification?
+      return idv_gpo_url if gpo_mail_bounced?
+      return idv_gpo_verify_url
+    end
+    return backup_code_reminder_url if user_needs_backup_code_reminder?
```

Comment:
I think we'll need to put this behind a feature flag to avoid backwards incompatibility in the 50/50 deploy state. If a server on the new version redirects me to the new route, but the request is handled by the old version, I'll get a 404.